main.py

```diff
@@ -1,10 +1,52 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_dict = json.load(file)
+
+    for player_name, information in players_dict.items():
+
+        race_name = information["race"]["name"]
+        race_description = information["race"]["description"]
+        if not Race.objects.filter(name=race_name).exists():
+            Race.objects.create(
+                name=race_name,
+                description=race_description
+            )
```

Comment:
Instead of this use [get_or_create](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create)